deploy/kubernetes/overlays/alpha/kustomization.yaml

```diff
@@ -1,17 +1,4 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-bases:
-- ../stable
-patches:
-- controller_add_snapshotter.yaml
-- controller_add_resizer.yaml
-patchesJson6902:
-- target:
-    group: rbac.authorization.k8s.io
-    version: v1
-    kind: ClusterRole
-    name: external-provisioner-role
-  path: rbac_add_snapshots_to_provisioner.yaml
 resources:
-- rbac_add_snapshotter.yaml
-- rbac_add_resizer.yaml
+- controller
```

Comment:
this seems like a really weird kustomization file. It doesn't really say anything anymore and I have no idea what it does

Reply:
This is to compose 3 kustomizations together